inst/@sym/taylor.m

```diff
@@ -89,6 +89,9 @@
     % taylor(f, [x,y], [a,b], 'param', ...)
     x = varargin{1};
     a = varargin{2};
+    if length(a) ~= length(x) && length(a) == 1
```

Comment:
this happens if `length(x) == `1 and `length(a) ~= 1`...?

Should it be: `if (length(a) == 1) && (length(x) > 1)`?